File: Modul 3/c/PRAK304-2310817110007-MuhammadDaffaMusyafa.c
#include <stdio.h>

int main(){
    int b;
    scanf("%d", &b);
    if (b == 0)
    {
       printf("nol");
    }
    else if (b >= 1 && b <= 9)
    {
        printf("satuan");
    }
    else if (b >= 10 && b <= 19)
    {
        printf("puluhan");
    }
    else if (b >= 20 && b <= 99)
    {
        printf("puluhan");
    }
    else if (b >= 100)
    {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    
    return 0;
}

File: Modul 3/py/PRAK304-2310817110007-MuhammadDaffaMusyafa.py
num = int(input())
if num >= 100:
    print("anda menginput lebih dari batas limit")
elif num < 100 and num >= 20:
    print("puluhan")
elif num >= 10 and num <= 19:
    print("belasan")
elif num < 10 and num >= 1:
    print("satuan")
elif num == 0:
    print("nol")